core/dbt/node_types.py

```diff
@@ -46,7 +46,8 @@ def documentable(cls) -> List['NodeType']:
             cls.Source,
             cls.Macro,
             cls.Analysis,
-            cls.Exposure
+            cls.Exposure,
+            cls.Test
```

Comment:
Confirmed that both types of tests (generic + bespoke) can use `doc` blocks in their descriptions, nice!